test/controllers/content_items_controller_test.rb

```diff
@@ -75,6 +75,43 @@ class ContentItemsControllerTest < ActionController::TestCase
     assert_response :forbidden
   end
 
+  test "honours AB Testing cookie" do
+    assert_equal "yes", ENV['ENABLE_NEW_NAVIGATION']
```

Comment:
Should the testing of the AB feature be more generic, ie not be tied to this specific test?